src/main/java/edu/cornell/kfs/sys/service/impl/BatchFileDirectoryServiceImpl.java

```diff
@@ -11,9 +14,28 @@
 
 public class BatchFileDirectoryServiceImpl implements BatchFileDirectoryService {
 
+    private static final Logger LOG = Logger.getLogger(BatchFileDirectoryServiceImpl.class);
+
+    @Override
+    public List<KeyValue> buildBatchFileStagingDirectories() {
+        LOG.debug("Starting BatchFileDirectoryServiceImpl.buildStagingDirectories().");
+        List<KeyValue> directoryKeyValueList = buildDirectoryKeyValuesList(CuBatchFileUtils.retrieveBatchFileStagingRootDirectories());
+        LOG.debug("Finished BatchFileDirectoryServiceImpl.buildStagingDirectories().");
+
+        return directoryKeyValueList;
+    }
+
     @Override
-    public List<KeyValue> buildDirectoryKeyValuesList(List<File> rootDirectories) {
-        List<KeyValue> keyValues = new ArrayList<KeyValue>();
+    public List<KeyValue> buildBatchFileLookupDirectories() {
+        LOG.debug("Starting BatchFileDirectoryServiceImpl.buildStagingAndReportsDirectories().");
+        List<KeyValue> directoryKeyValueList = buildDirectoryKeyValuesList(BatchFileUtils.retrieveBatchFileLookupRootDirectories());
+        LOG.debug("Finished BatchFileDirectoryServiceImpl.buildStagingAndReportsDirectories().");
```

Comment:
Another area where the strings are using one name but the method is using another name.